src/CModel.cc

```diff
@@ -61,11 +63,18 @@ PTR(afw::detection::Footprint) _mergeFootprints(
         afw::detection::Threshold(0x1, afw::detection::Threshold::BITMASK),
         1 // npixMin
     );
+    afw::geom::Point2I pixel(center);
     if (fpSet.getFootprints()->size() > 1u) {
+        result.setFlag(CModelResult::INCOMPLETE_FIT_REGION, true);
+        for (std::size_t n = 0; n < fpSet.getFootprints()->size(); ++n) {
+            if ((*fpSet.getFootprints())[n]->contains(pixel)) {
```

Comment:
A good idea, but I'd rather not implement it yet, as doing it right will require use of `auto` and the HSC side doesn't yet build with C++11 features enabled.  Until then (or until we've fully completely the HSC merge) I'd rather keep this code in sync with the HSC side.

We'll have to touch this code anyway sometime on DM-3559 (which will necessarily happen after the HSC merge is complete), so I'll plan to make the change there.